javascript3/week2/script.js

```diff
@@ -0,0 +1,372 @@
+const apiURL =
+  "https://raw.githubusercontent.com/GayathriVenkatraman/gayathrivenkatraman.github.io/refs/heads/main/recipeApp.json";
+let recipes = [];
+async function fetchRecipes() {
```

Comment:
I can see this method is doing 2 things, it's fetching and displaying data, so in order to make it more readable I would suggest to call fetchAndDisplayData()